app/views/shared/_country_list.html.erb

```diff
@@ -9,10 +9,24 @@
             <svg xmlns="http://www.w3.org/2000/svg" class="list--countries-icon svg-icon svg-icon--country" focusable="false">
               <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--<%= country[:region] %>"></use>
             </svg>
-            <%= link_to "#{country[:text]}", "#{country[:url]}", class: 'list--countries-link' %>
+            <% if region_uk?(country[:region]) %>
+              <%= link_to( 
+                  country[:text],
+                  uk_strategy_path(id: country[:slug], locale: 'en'),
+                  class: 'list--countries-link'
+              )%>
+              <% else %>
+                <%= link_to( 
+                    country[:text],
+                    regional_strategy_path(id: country[:slug], locale: 'en'),
+                    class: 'list--countries-link'
+                )%>
+            <% end %>
```

Comment:
The only difference between the two conditional branches is literally the path helper name – this made me thing that one way to DRY this is to have:

```ruby
<%= link_to(
    country[:text],
    strategy_path(country),
    class: 'list--countries-link'
)%>
```

where `strategy_path` could be an helper defined like:

```ruby
def strategy_path(country)
  if country[:region] == 'uk'
    uk_strategy_path(id: country[:slug], locale: 'en')
  else
    regional_strategy_path(id: country[:slug], locale: 'en')
  end
end
```